content/blog/2022/01/mutating-policy-behave-as-validating.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Kubewarden has policies to cover all the Kubernetes Pod Security Policies.
+authors:
+	- José Guilherme Vanz
+date: 2022-01-31
+---
+
+We at the Kubewarden team has been doing an effort to allow our users to have
+policies equivalent for all the deprecated Pod Security Policies (PSP).
+So, we wrote policies to replicate the same validations available in the Kubernetes
+PSPs. During this work, we counted on the community help to map and validate
+the policies.
```

Comment:
```suggestion
The Kubewarden team worked tirelessly to create equivalent Kubewarden policies for all the deprecated Pod Security Policies (PSP). In order to reach this very important milestone, the team wrote the policies with the same validations available in the Kubernetes PSPs, and we counted on the community help to map and validate the policies.

This will allow our users to continue using the policies they have enforced in their Kubernetes clusters, and avoid any downtimes or security issues.
```

Reply:
While the first paragraph you might take it as is, the second one can be modified to emphasize which real advantages the users have by using Kubewarden policies and which risks they mitigate (I put two examples).